packages/react/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -25,8 +25,10 @@ const Breadcrumb = ({
   });
 
   return (
-    <nav className={className} {...rest}>
-      {children}
+    <nav aria-label="Breadcrumb">
+      <ol className={className} {...rest}>
```

Comment:
We should be able to remove `{..rest}` from here and only specify `ol` class name here 👀